1.20/s2i/bin/usage

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 cat <<EOF
-This is a S2I Rust 2.0 base image. To use it, install S2I: https://github.com/openshift/source-to-image
+This is a S2I Rust 1.20 base image. To use it, install S2I: https://github.com/openshift/source-to-image
```

Comment:
I'm *assuming* this was just a typo, so I didn't think it deserved a separate PR, but I'll break it out if you prefer...

Reply:
Yes, that's a typo. Thank you for correcting it.